src/com/rcv/CVRReader.java

```diff
@@ -125,7 +125,7 @@ public void parseCVRFile(
         rankings.add(ranking);
       }
 
-      CastVoteRecord cvr = new CastVoteRecord(ballotID, rankings, fullCVRData);
+      CastVoteRecord cvr = new CastVoteRecord(excelFilePath, ballotID, rankings, fullCVRData);
```

Comment:
Yeah, I like the idea of just parsing out the filename instead of using the whole path.

Reply:
ok